### PR TITLE
Update Pronouns admin settings 

### PR DESCRIPTION
**Pull request by aymen-ms**

### CODE REVIEW

File: api-reference/beta/resources/peoplesetting.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: "peopleSetting resource type"
```
Comment (antonsegh) [inline api-reference/beta/resources/peoplesetting.md:2]: is it peopleSetting or peopleSettingS ?

also havent review said 'peopleAdminSettings' ?

Comment (aymen-ms) [reply api-reference/beta/resources/peoplesetting.md]: updated the resource name

**Comment by dodaromike** [comment]
@aymen-ms Please resolve conflicts in this PR so we can proceed with review.

**Review by dodaromike — APPROVED** [review]

File: concepts/permissions-reference.md [hunk 2]
```diff
@@ -1829,6 +1829,24 @@ For more complex scenarios involving multiple permissions, see [Permission scena
 
 ---
 
+## People Settings permissions
```
Comment (Danielabom) [inline concepts/permissions-reference.md:1832]:
```suggestion
## People settings permissions
```

File: changelog/Microsoft.People.json [hunk 3]
```diff
@@ -130,45 +130,87 @@
...
 			"WorkloadArea": "Identity and access",
 			"SubArea": "Directory management"
-		  }
+		},
+		{
+			"ChangeList": [
+				{
+					"Id": "3d55601a-1632-4115-9e52-4d8a174040d4",
+					"ApiChange": "Resource",
+					"ChangedApiName": "peopleSetting",
```
Comment (Danielabom) [inline changelog/Microsoft.People.json:178]:
```suggestion
					"ChangedApiName": "peopleAdminSetting",
```

File: api-reference/beta/resources/peopleadminsettings.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
+---
+title: "peopleAdminSettings resource type"
+description: "Represents a setting to control the people related admin settings in the tenant"
```
Comment (Danielabom) [inline api-reference/beta/resources/peopleadminsettings.md:3]:
```suggestion
description: "Represents a setting to control the people-related admin settings in the tenant"
```

File: changelog/Microsoft.People.json [hunk 5]
```diff
@@ -130,45 +130,87 @@
...
+					"ChangeType": "Addition",
+					"Description": "Added the [peopleAdminSettings](https://learn.microsoft.com/en-us/graph/api/resources/peopleadminsettings?view=graph-rest-beta) resource",
+					"Target": "peopleAdminSettings"
+				},
+				{
+					"Id": "3d55601a-1632-4115-9e52-4d8a174040d4",
+					"ApiChange": "Relationship",
+					"ChangedApiName": "pronounsSettings",
+					"ChangeType": "Change",
+					"Description": "Moved the **pronounsSettings** relationship to [peopleAdminSettings](https://learn.microsoft.com/en-us/graph/api/resources/peopleadminsettings?view=graph-rest-beta) resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.People.json:188]: For this change, you need three entries:
1. To track that you are deleting a nav property from a resource. 
2. To track that you are adding a nav property to a resource.
3.  To show that you are renaming a nav property.

File: changelog/Microsoft.People.json [hunk 6]
```diff
@@ -130,45 +130,87 @@
...
+					"Target": "organizationSettings"
+				},
+				{
+					"Id": "3d55601a-1632-4115-9e52-4d8a174040d4",
+					"ApiChange": "Permission",
+					"ChangedApiName": "PeopleSettings.Read.All",
+					"ChangeType": "Addition",
+					"Description": "Added support for the policy-based application permission of `PeopleSettings.Read.All` to read tenant-wide [peoplesettings](https://learn.microsoft.com/en-us/graph/api/resources/peopleadminsettings?view=graph-rest-beta) resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.People.json:196]:
```suggestion
					"Description": "Added support for the policy-based application permission of `PeopleSettings.Read.All` to read tenant-wide operations of the [peopleSettings](https://learn.microsoft.com/en-us/graph/api/resources/peopleadminsettings?view=graph-rest-beta) resource.",
```

File: changelog/Microsoft.People.json [hunk 7]
```diff
@@ -130,45 +130,87 @@
...
+					"Target": "PeopleSettings"
+				},
+				{
+					"Id": "3d55601a-1632-4115-9e52-4d8a174040d4",
+					"ApiChange": "Permission",
+					"ChangedApiName": "PeopleSettings.ReadWrite.All",
+					"ChangeType": "Addition",
+					"Description": "Added support for the policy-based application permission of `PeopleSettings.Read.All` to the read and write tenant-wide [peoplesettings](https://learn.microsoft.com/en-us/graph/api/resources/peopleadminsettings?view=graph-rest-beta) resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.People.json:204]:
```suggestion
					"Description": "Added support for the policy-based application permission of `PeopleSettings.Read.All` to the read and write tenant-wide operations of the [peoplesettings](https://learn.microsoft.com/en-us/graph/api/resources/peopleadminsettings?view=graph-rest-beta) resource.",
```

File: concepts/permissions-reference.md [hunk 8]
```diff
@@ -1829,6 +1829,24 @@ For more complex scenarios involving multiple permissions, see [Permission scena
...
 ---
 
+## People settings permissions
+
+#### Delegated permissions
+
+|   Permission    |  Display String   |  Description | Admin Consent Required | Microsoft Account supported |
+|:----------------|:------------------|:-------------|:-----------------------|:--------------|
+| _PeopleSettings.Read.All_ | Read tenant-wide people settings | Allows the application to read tenant-wide people settings, on behalf of the signed-in user. | Yes | No |
+| _PeopleSettings.ReadWrite.All_ | Read and write tenant-wide people settings | Allows the application to read and write tenant-wide people settings, on behalf of the signed-in user. | Yes | No |
```
Comment (Danielabom) [inline concepts/permissions-reference.md:1839]:
```suggestion
| _PeopleSettings.ReadWrite.All_ | Read and write tenant-wide people settings | Allows the application to read and write tenant-wide people settings on behalf of the signed-in user. | Yes | No |
```

File: concepts/permissions-reference.md [hunk 9]
```diff
@@ -1829,6 +1829,24 @@ For more complex scenarios involving multiple permissions, see [Permission scena
 
 ---
 
+## People settings permissions
+
+#### Delegated permissions
+
+|   Permission    |  Display String   |  Description | Admin Consent Required | Microsoft Account supported |
+|:----------------|:------------------|:-------------|:-----------------------|:--------------|
+| _PeopleSettings.Read.All_ | Read tenant-wide people settings | Allows the application to read tenant-wide people settings, on behalf of the signed-in user. | Yes | No |
```
Comment (Danielabom) [inline concepts/permissions-reference.md:1838]:
```suggestion
| _PeopleSettings.Read.All_ | Read tenant-wide people settings | Allows the application to read tenant-wide people settings on behalf of the signed-in user. | Yes | No |
```

File: concepts/permissions-reference.md [hunk 10]
```diff
@@ -1829,6 +1829,24 @@ For more complex scenarios involving multiple permissions, see [Permission scena
...
+| _PeopleSettings.ReadWrite.All_ | Read and write tenant-wide people settings | Allows the application to read and write tenant-wide people settings, on behalf of the signed-in user. | Yes | No |
+
+#### Application permissions
+
+|   Permission    |  Display String   |  Description | Admin Consent Required |
+|:-----------------------------|:-----------------------------------------|:-----------------|:-----------------|
+| _PeopleSettings.Read.All_ | Read all tenant-wide people settings | Allows the application to read tenant-wide people settings, without a signed-in user. | Yes |
```
Comment (Danielabom) [inline concepts/permissions-reference.md:1845]:
```suggestion
| _PeopleSettings.Read.All_ | Read all tenant-wide people settings | Allows the application to read tenant-wide people settings without a signed-in user. | Yes |
```

File: concepts/permissions-reference.md [hunk 11]
```diff
@@ -1829,6 +1829,24 @@ For more complex scenarios involving multiple permissions, see [Permission scena
...
+| _PeopleSettings.ReadWrite.All_ | Read and write all tenant-wide people settings | Allows the application to read and write tenant-wide people settings, without a signed-in user. | Yes |
```
Comment (Danielabom) [inline concepts/permissions-reference.md:1846]:
```suggestion
| _PeopleSettings.ReadWrite.All_ | Read and write all tenant-wide people settings | Allows the application to read and write tenant-wide people settings without a signed-in user. | Yes |
```

File: changelog/Microsoft.People.json [hunk 12]
```diff
@@ -130,45 +130,95 @@
...
+			"Id": "3d55601a-1632-4115-9e52-4d8a174040d4",
+			"Cloud": "Prod",
+			"Version": "beta",
+			"CreatedDateTime": "2023-05-23T16:27:03.4918335Z",
+			"WorkloadArea": "People and workplace intelligence",
+			"SubArea": "People"
+		}
 	]
-}
+}
```
Comment (Danielabom) [inline changelog/Microsoft.People.json:224]: Please add a changelog entry to track that you deleted an operation from a resource.

File: changelog/Microsoft.People.json [hunk 13]
```diff
@@ -130,45 +130,95 @@
...
+					"ChangeType": "Addition",
+					"Description": "Added the [peopleAdminSettings](https://learn.microsoft.com/en-us/graph/api/resources/peopleadminsettings?view=graph-rest-beta) resource",
+					"Target": "peopleAdminSettings"
+				},
+				{
+					"Id": "3d55601a-1632-4115-9e52-4d8a174040d4",
+					"ApiChange": "Relationship",
+					"ChangedApiName": "pronounsSettings",
+					"ChangeType": "Deletion",
+					"Description": "Removed the **pronounsSettings** relationship from [organizationSettings](https://learn.microsoft.com/en-us/graph/api/resources/organizationSettings?view=graph-rest-beta) resource.",
```
Comment (Danielabom) [inline changelog/Microsoft.People.json:188]:
```suggestion
					"Description": "Removed the **pronounsSettings** relationship from the [organizationSettings](https://learn.microsoft.com/en-us/graph/api/resources/organizationSettings?view=graph-rest-beta) resource.",
```

File: changelog/Microsoft.People.json [hunk 14]
```diff
@@ -130,45 +130,95 @@
...
+					"Target": "PeopleSettings"
+				}
+			],
+			"Id": "3d55601a-1632-4115-9e52-4d8a174040d4",
+			"Cloud": "Prod",
+			"Version": "beta",
+			"CreatedDateTime": "2023-05-23T16:27:03.4918335Z",
+			"WorkloadArea": "People and workplace intelligence",
+			"SubArea": "People"
+		}
```
Comment (Danielabom) [inline changelog/Microsoft.People.json:222]: Please add the entries for delegated permissions.

File: api-reference/beta/resources/peopleadminsettings.md [hunk 15]
```diff
@@ -0,0 +1,49 @@
+---
+title: "peopleAdminSettings resource type"
```
Comment (Danielabom) [inline api-reference/beta/resources/peopleadminsettings.md:2]: Please add this resource to the TOC, and its operations.

File: changelog/Microsoft.People.json [hunk 16]
```diff
@@ -130,45 +130,95 @@
...
+					"ChangeType": "Addition",
+					"Description": "Added the [list](https://learn.microsoft.com/en-us/graph/api/organizationsettings-list-pronounssettings?view=graph-rest-beta&tabs=http) operation to the [pronounsSettings](https://learn.microsoft.com/en-us/graph/api/resources/pronounssettings?view=graph-rest-beta) resource, to get the tenant-level settings for pronouns",
+					"Target": "pronounsSettings"
+				},
+				{
+					"Id": "827d51a9-63d6-4d06-a157-262bf134e8d8",
+					"ApiChange": "Method",
+					"ChangedApiName": "PATCH",
+					"ChangeType": "Addition",
+					"Description": "Added the [udpate](https://learn.microsoft.com/en-us/graph/api/pronounssettings-update?view=graph-rest-beta&tabs=http) operation to the [pronounsSettings](https://learn.microsoft.com/en-us/graph/api/resources/pronounssettings?view=graph-rest-beta) resource, to update the tenant-level settings for pronouns",
```
Comment (Danielabom) [inline changelog/Microsoft.People.json:154]:
```suggestion
					"Description": "Added the [udpate](https://learn.microsoft.com/en-us/graph/api/pronounssettings-update?view=graph-rest-beta&tabs=http) operation to the [pronounsSettings](https://learn.microsoft.com/en-us/graph/api/resources/pronounssettings?view=graph-rest-beta) resource, to update the tenant-level settings for pronouns",
```
```suggestion
					"Description": "Added the [udpate](https://learn.microsoft.com/en-us/graph/api/pronounssettings-update?view=graph-rest-beta&tabs=http) operation to the [pronounsSettings](https://learn.microsoft.com/en-us/graph/api/resources/pronounssettings?view=graph-rest-beta) resource",
```

**Review by Danielabom — CHANGES_REQUESTED** [review]
@aymen-ms, all my comments are in. Please review them. Thanks!

File: api-reference/beta/api/peopleadminsettings-list-pronouns.md [hunk 17]
```diff
@@ -0,0 +1,78 @@
+---
+title: "List pronounsSettings"
```
Comment (Danielabom) [inline api-reference/beta/api/peopleadminsettings-list-pronouns.md:2]: Please make you add a changelog entry to delete this method.

File: api-reference/beta/toc.yml [hunk 18]
```diff
@@ -12104,6 +12096,17 @@ items:
             href: api/onpremisesdirectorysynchronization-get.md
           - name: Update
             href: api/onpremisesdirectorysynchronization-update.md
+        - name: People Admin Settings (preview)
```
Comment (Danielabom) [inline api-reference/beta/toc.yml:12099]:
```suggestion
        - name: People admin settings (preview)
```

**Review by Danielabom — APPROVED** [review]
Thank you for addressing my comments @aymen-ms. PR approved.

**Pull request merged**